src/app/pages/list/multiselect-list/multiselect-list.component.spec.ts

```diff
@@ -21,19 +21,4 @@ describe('MultiselectListComponent', () => {
     it('should create', () => {
         void expect(component).toBeTruthy();
     });
-
```

Comment:
We should replicate these tests in the new service instead of just deleting them.

Reply:
I've added tests for the new service.